history/cultures/pandaren_group.txt

```diff
@@ -1,45 +1,64 @@
+# Mainland Pandaren (Sheili)
+# Wandering Pandaren (Shen-zin)
+
 1.1.1 = {
-	discover_innovation = innovation_mustering_grounds 
-	#
-	discover_innovation = innovation_casus_belli
-	discover_innovation = innovation_currency_01 
+	discover_innovation = innovation_quilted_armor 
```

Comment:
Pandarens don't have aerial.
Moreover, they should have Flying Beats (serpents).
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/127783067-95174ec9-0d7c-4d91-acab-861e44592248.png)

</details>

Reply:
> Pandarens don't have aerial.
> Moreover, they should have Flying Beats (serpents).

I see the aerial tech, but what about the flying beast? They will be done on another branch later?

Reply:
Do you mean the serpents? Like cultrual maa?

Reply:
Do you specifically mean the massive aerial cultural maa? For the serpents you mentioned?

Reply:
> Do you specifically mean the massive aerial cultural maa? For the serpents you mentioned?

Yes.

Reply:
Yes I will add them to cultural maa